.github/workflows/release.yml

```
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Release Tag (e.g. 6.0.0)"
```

Suggest putting the "v" in the example to make it clear what the tag will be

```suggestion
        description: "Release Tag (e.g. v6.0.0)"
```

---

Nevermind, I see that the existing convention is w/o the `v` prefix

---

Yes! I'll make this clearer in the description, that was why I gave the example w/o `v` - Python versions/tags do NOT want it as a prefix 😬